medusa/service/grpc/medusa.proto

```diff
@@ -71,11 +73,22 @@ message BackupSummary {
   int32               finishedNodes = 5;
   repeated BackupNode nodes = 6;
   StatusType    status = 7;
+  string    backupType = 8;
 }
 
 message BackupNode {
   string         host = 1;
   repeated int64 tokens = 2;
   string         datacenter = 3;
   string         rack = 4;
+}
+
+message PurgeBackupsRequest {
```

Comment:
Do we need to think about supplying any type of identity for the backup request caller?  For example, if we needed to track in the logs the originator of the purge request.  In our case, for now, this would be something like `medusa-operator` or `k8ssandra-operator`.  This could be useful for correlation with logging across systems.

Reply:
I'm not sure I see how this additional info could be useful 🤔  
The reality is that this call will only be implemented in k8ssandra-operator, so we already know where the calls are coming from.

Reply:
I'm 🆗 with not adding it in this case.  

It becomes extremely helpful for logging correlation along with date-time completion times when logged.